CONTRIBUTING.md

```diff
@@ -42,5 +49,41 @@ Included in this repo is the file `helphub.wordpress.2017-06-15.xml`. Import thi
 6. Click the button to begin. It may take a while to complete. If there are some failed imports, it should still be okay.
 7. Your local install of HelpHub should now be ready to view in your browser.
 
+Once you have Helphub in your localhost up and running, you can contribute with pull requests either from your own fork or directly in this repository, after you're added to it as a contributor. We are using [Travis CL](https://travis-ci.org/) for tests on every pull request. You can, also, run these tests localy before pushing your code (more on this later). Development covers work on both, theme and plugins and requires following [best practices](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) and [WordPress coding standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards).
+
+#### Helphub Theme
+
+**Requirements:**
+
+- [npm](https://www.npmjs.com/get-npm)
+- [gulp](https://gulpjs.com/)
+- [SASS](http://sass-lang.com/)
+
+Helphub theme uses [SASS](http://sass-lang.com/) for applying styles as it provides possibility for breaking one large style.css file into smaller partials and, therefore, reduce possible GIT conflicts caused with multiple modifications of the same file in different branches. Maintenance is, also, greatly improved. Once modified SASS (.scss) files need to be compiled into style.css. Compiler of our choice is [gulp](https://gulpjs.com/). To be able to run gulp you need [npm](https://www.npmjs.com/get-npm). We will assume that you already have npm.
+
+In terminal navigate to theme folder. First we need to install dependencies from `package.js` file:
+```
+npm install
+```
+
+After this command has ran in your terminal you'll have another folder in theme root, `node_modules`. This folder is ignored in .gitignore and contains all the tools we need for running gulp tasks, defined in `gulpfile.js`. This file contains automated tasks for the project (which can be anything - building svg sprites, minifying css and js files etc). We are using it for two tasks: `css` and `watch`.
```

Comment:
• After this command has run in your terminal you'll have another folder in the theme root, `node_modules`. This folder is ignored in `.gitignore` and contains all the tools we need for running gulp tasks, defined in `gulpfile.js`.